common/build.gradle

```diff
@@ -37,8 +37,9 @@ android {
 
 //        buildConfigField "String", "BITCOIN_FAUCET", "\"http://134.122.59.107:3000\""
         buildConfigField "String", "BITCOIN_FAUCET", "\"https://taproot.tribler.org\""
-//        buildConfigField "String", "BITCOIN_DEFAULT_PEER", "\"134.122.59.107\""
-        buildConfigField "String", "BITCOIN_DEFAULT_PEER", "\"131.180.27.224\""
+        // 10.0.2.2 - IP address of the host system on Android emulator
+        // Refer to included Docker Compose to launch the nodes locally
+        buildConfigField "String", "BITCOIN_DEFAULT_PEER", "\"10.0.2.2\""
```

Comment:
Again local IP

Reply:
I've replaced the IP address with the old one and left our line commented out for local development (it's mentioned in README that it has to be configured in advance).